### PR TITLE
Incentive module multiple reward denom tests

### PR DESCRIPTION
**Pull request by denalimarsh**
Updates the Incentive module's test suite for multiple reward denoms.

### CODE REVIEW

File: x/incentive/keeper/payout_test.go [hunk 1]
```diff
@@ -153,179 +154,189 @@ func (suite *KeeperTestSuite) TestPayoutUSDXMintingClaim() {
...
+				expectedRewards:          cs(c("hard", 21142771200)), // 10571385600 (deposit reward) + 10571385600 (borrow reward)
+				expectedPeriods:          vesting.Periods{vesting.Period{Length: 32918400, Amount: cs(c("hard", 21142771200))}},
+				isPeriodicVestingAccount: true,
+			},
+			errArgs{
+				expectPass: true,
+				contains:   "",
+			},
+		},
+		{
```
Comment (karzak) [inline x/incentive/keeper/payout_test.go:199]: Add test with multiple coins on the claim, add similar test for USDX minting rewards

File: x/incentive/keeper/rewards.go [hunk 2]
```diff
@@ -521,6 +523,7 @@ func (k Keeper) SynchronizeHardDelegatorRewards(ctx sdk.Context, delegator sdk.A
 	}
 
 	// Add rewards to delegator's hard claim
+	// TODO: types.HardLiquidityRewardDenom
```
Comment (karzak) [inline x/incentive/keeper/rewards.go:526]: merge artifact I think- can delete

**Review by karzak — CHANGES_REQUESTED** [review]
Looks really good.

Add multiple coins payout tests, review merge changes, otherwise LGTM. Feel free to merge once new tests are passing and added.

File: x/incentive/keeper/rewards.go (hunk 2 above)
Comment (denalimarsh) [reply x/incentive/keeper/rewards.go]: Removed in https://github.com/Kava-Labs/kava/pull/788/commits/8d1cc4a9f8a912b2cfec44b1803c3aaa93c6ff57

File: x/incentive/keeper/payout_test.go (hunk 1 above)
Comment (denalimarsh) [reply x/incentive/keeper/payout_test.go:199]: Updated payout test in https://github.com/Kava-Labs/kava/pull/788/commits/c63ff89829ca9b789374f3999e0a928104020ecd.

What do you mean by 'similar test for USDX minting rewards'? USDXMintingClaims don't use `MultiRewardPeriod`/`MultiRewardIndex` yet. If the suggestion here is to test multiple reward coins within the current set up (e.g. USDXMinting RewardPeriods params contain two RewardPeriods for bnb/xrp that have different reward denoms such as ukava/hard) I'd rather just update USDXMintingClaims to use `MultiRewardPeriods` and then test multiple reward denoms using the existing testing process from TestPayoutHardLiquidityProviderClaim. I can make an issue for it.

**Comment by denalimarsh** [comment]
A bunch of tests and some functionality got deleted by the master merge so I added them back and updated the master tests in https://github.com/Kava-Labs/kava/pull/788/commits/bfaa1b2f2a63bf80c01432d0e93687345586e616. Hopefully nothing else got accidentally removed, currently the tests are passing and everything seems to be working.

File: x/incentive/keeper/payout_test.go (hunk 1 above)
Comment (karzak) [reply x/incentive/keeper/payout_test.go:199]: Ah my bad, forgot we didn't implement Multi rewards for USDX minting yet. Ignore.

**Review by karzak — APPROVED** [review]

**Pull request merged**